### DISCUSSION TITLE
How show the view of enable notifications above of bottom navigation?

### DISCUSSION THREAD

**Discussion by LJMartinez07** [lukebrandonfarrell/react-native-navigation-drawer-extension#89] [Q&A] [answered]
I guys, I was looking at this library and I saw the following global modal in the example screen and I want to do this on my app but I don't know how can I achieve it.

![Screen Shot 2022-09-23 at 10 34 22 AM](https://user-images.githubusercontent.com/33770223/191985901-520777f9-c4b5-4797-9007-63e2e22cc290.png)

**Comment by LJMartinez07** [comment, marked as answer]
For those who ask how to achieve this, I did it with [react-native-modal](https://github.com/react-native-modal/react-native-modal) as they show in the example of the bottom half.

**Reply by lukebrandonfarrell** [reply]
Setting the `direction` prop to `bottom` should also do this without `react-native-modal`
